woocommerce/woocommerce
# Add/36019 php product form helper classes

### All Submissions:






### Changes proposed in this Pull Request:

Adds some of the initial ProductForm classes for 3PD's to extend the new Product form through PHP.
This currently only adds the ability to add fields.

Closes #36019






### How to test the changes in this Pull Request:



1. Run PHP unit tests locally to see if they all pass, see https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/tests/README.md#initial-setup for initial setup and how to.



### Other information:




### FOR PR REVIEWER ONLY:
